spectrum/test_article.py

```diff
@@ -130,6 +130,19 @@ def test_article_with_unicode_content(generate_article):
     journal_page = checks.JOURNAL.article(id=article.id(), has_figures=article.has_figures())
     assert "Szymon \xc5\x81\xc4\x99ski" in journal_page
 
+@pytest.mark.journal
+@pytest.mark.continuum
+@pytest.mark.bot
+@pytest.mark.lax
```

Comment:
Do these mean it'll run for those various subsets? If so, can probably limit to Journal?

Reply:
These are used for filtering, so only `journal` is needed, yes